DVR/Session.swift

```diff
@@ -195,6 +197,10 @@ public class Session: NSURLSession {
 			}
         }
 
+        var cassetteName = "cassette"
+        if let s = cassetteURL?.lastPathComponent {
+            cassetteName = s.substringToIndex(s.endIndex.advancedBy(-5))
```

Comment:
I think it would be much better to get the length directly instead of guessing it's 5